### PR TITLE
Story3 create an order

### PR DESCRIPTION
**Pull request by ghost**
Hello Fatima & Richard,

This is Story3 completed. 

What is not included in this Story is the following tasks:
1) No active form validation yet.
2) No POST request to API.
3) No confirmation modal created yet.

These three items will be completed with Story7.

What needs to be done to merge this with other Stories:
1) Main header nav bar needs to be included in Order page.
2) Font + any other styling needs to be applied to be consistent with home page.
3) Add menu card items to the Order page.
4) Cancel Order button in OrderSummary modal needs to take you home page.
5) Cancel Order button in Order needs to take you to home page.

I am happy to complete these once this is merged to main and I can access the home page.

### CODE REVIEW

File: src/Components/Order/OrderSummary/index.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+}
+
+code {
+    font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
+    monospace;
+}
+
+
+.modal-header, .modal-body, #orderSummaryFooter {
+    color: white;
```
Comment (RBingham99) [inline src/Components/Order/OrderSummary/index.css:17]: make hex code please

File: src/Components/Order/OrderSummary/index.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
+import 'bootstrap/dist/css/bootstrap.min.css';
+import 'bootstrap/dist/js/bootstrap.bundle.min';
+import './index.css';
+
+
+
```
Comment (RBingham99) [inline src/Components/Order/OrderSummary/index.js:6]: to many blank lines

File: src/Components/Order/index.css [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    display: flex;
+    gap: 2rem;
+    margin: 0 0 0 2rem;
+}
+
+.orderTitles {
+    margin: 0 0 0 3rem;
+}
+
+
```
Comment (RBingham99) [inline src/Components/Order/index.css:31]: to many blank lines

File: src/Components/Order/index.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
+import 'bootstrap/dist/css/bootstrap.min.css';
+import 'bootstrap/dist/js/bootstrap.bundle.min';
+import './OrderSummary';
+import AnchorLink from "react-anchor-link-smooth-scroll";
+// import './Navbar';
+import OrderSummary from "./OrderSummary";
+import './index.css';
+
+
```
Comment (RBingham99) [inline src/Components/Order/index.js:9]: to many blank lines here and on lines 11/12

**Review by RBingham99 — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
All changes made.

**Review by RBingham99 — APPROVED** [review]

**Pull request merged**